openstack/identity/v3/tokens/requests.go

```diff
@@ -182,7 +182,7 @@ func Get(c *gophercloud.ServiceClient, token string) (r GetResult) {
 func Validate(c *gophercloud.ServiceClient, token string) (bool, error) {
 	resp, err := c.Request("HEAD", tokenURL(c), &gophercloud.RequestOpts{
 		MoreHeaders: subjectTokenHeaders(c, token),
-		OkCodes:     []int{200, 204, 400, 401, 403, 404},
+		OkCodes:     []int{200, 204, 400, 403, 404},
```

Comment:
While we're on the topic (but I can open a separate Issue/PR), the only other place I see 400 codes listed as "OK" are here:

https://github.com/gophercloud/gophercloud/blob/c5c590a7ee87267c978586fdd1655db0dd4b6e05/openstack/imageservice/v2/members/requests.go#L21

Should those be removed, too?

To confirm, an "OK Code" is one that would only signify a successful action? Not one that is necessarily used in the API service itself?